cartridges/int_algolia_sfra/cartridge/controllers/Search.js

```diff
@@ -48,6 +48,9 @@ server.replace('Show', cache.applyShortPromotionSensitiveCache, consentTracking.
             var hits;
             var contentHits;
             var queryHits;
+            //For Algolia, we don't need personalized cache, as the real results are fetched on the front-end.
+            res.cachePeriod = 24; // eslint-disable-line no-param-reassign
+            res.cachePeriodUnit = 'hours'; // eslint-disable-line no-param-reassign
```

Comment:
Would it work if we override the parameter above instead?
```
server.replace('Show', cache.applyDefaultCache, ...
```

Reply:
I thought it, but in this case, we will also change the logic when Algolia is disabled (I mean they can disable Algolia but keep cartridge).

Reply:
Ok, this use case makes sense :+1: